source-postgres/discovery.go

```diff
@@ -250,8 +257,45 @@ func translateRecordField(column *sqlcapture.ColumnInfo, val interface{}) (inter
 			fmt.Fprintf(s, "%02x", x[i])
 		}
 		return s.String(), nil
+	case string:
+		if len(x) > truncateColumnThreshold {
```

Comment:
Would it be practical to add some indication that these fields have been truncated if they do get truncated? It will be very obvious for some types from the output being `{"flow_truncated":true,"original_size":%d}` but for ex. a string that gets truncated won't have that kind of indication from what I can tell.

Reply:
Yeah, we can implement pretty much whatever transformation we want here, I just went with what was described to me in the initial feature request. Do you have anything specific in mind?